packages/nimble-components/src/table-column/base/models/column-internals.ts

```diff
@@ -105,13 +115,13 @@ export class ColumnInternals<
     /**
      * Template for the group header view
      */
-    public readonly groupHeaderViewTemplate: ViewTemplate<TableGroupRow>;
+    public readonly groupHeaderViewTemplate?: ViewTemplate<TableGroupRow>;
 
     /**
      * Whether or not this column can be used to group rows by
      */
     @observable
-    public groupingDisabled = false;
+    public groupingDisabled = true;
```

Comment:
This seems significant to change the defaults of two properties. Is it not? If it is, what are the repercussions? I would presume this would have some amount of cascading implementation, but maybe not?

Reply:
Given our mixin pattern, I think it makes sense to disable grouping & sorting by default and only enable those features on a column if the column wants them enabled. The `mixinGroupableColumnAPI` has `groupingDisabled` and `mixinSortableColumnAPI` has `sortingDisabled`, both of which are initialized to `false`, which in turn sets these `ColumnInternals` properties to `false`. Therefore, I don't expect this change to have an impact on existing columns using grouping or sorting; it would just make a column not using those mixins (such as the menu-button column) have those features disabled by default.